File: go/FizzBuzz/fizzbuzz.go
package main

func main() {
	standardFizBuzz()
}

func standardFizBuzz() {
	for i := 1; i < 100; i++ {
		println(i)
		if i%3 == 0 {
			println("Fizz")
		}

		if i%5 == 0 {
			println("Buzz")
		}

		if i%3 == 0 && i%5 == 0 {
			println("FizzBuzz")
		}
	}
}

File: php/FizzBuzz.php
<?php

function fizzBuzz($number) {
	for ($i = 1; $i <= $number; $i++) {
		if ($i % 3 == 0 and $i % 5 == 0) {
			echo "FizzBuzz\n";
		}

		elseif ($i % 3 == 0) {
			echo "Fizz\n";
		}

		elseif ($i % 5 == 0) {
			echo "Buzz\n";
		}

		else {
			echo $i . "\n";
		}
	}
}

fizzBuzz(100);
?>
